app/models/host/base.rb

```diff
@@ -174,6 +174,8 @@ def set_interfaces(parser)
 
       parser.interfaces.each do |name, attributes|
         iface = get_interface_scope(name, attributes).try(:first) || interface_class(name).new(:managed => false)
+        # update bond when interface is attached to it
+        update_attached_interfaces(name, attributes) unless attributes['virtual']
```

Comment:
shouldn't this happen in `set_interface` just like `update_virtuals`? I suppose it should also happen only if identifier changed... maybe it should be combined with the `update_virtuals` method into single one